### PR TITLE
Front end/browse vehicles

### LINKED ISSUE #99: [SUBTASK of story #81] Implement a "Browse Vehicle" tab on the main navigation menu.
### :rocket: Summary
Implement a "Browse Vehicle" tab on the main navigation menu.

### :scroll: Description
Add a "Browse Vehicle" tab to the main navigation menu of the car rental application to provide users with easy access to explore available vehicles.

### :heavy_check_mark: Acceptance Criteria
- [ ] The main navigation menu includes a "Browse Vehicle" tab.
- [ ] Clicking on the "Browse Vehicle" tab redirects users to a page where they can view all available vehicles.
- [ ] Users can filter vehicles by make, model, year, and other relevant criteria.

### LINKED ISSUE #100: [SUBTASK of story #81] Display the inventory of vehicles as a list/grid with all the relevant information (manufacture, model, year, price per day, availability).

### LINKED ISSUE #101: [SUBTASK of story #81] Implement a filter function (in the search bar) based on the manufacture, price range, and availability dates.

### LINKED ISSUE #102: [SUBTASK of story #81] A detailed view of each vehicle with addition information (features, rental terms, special offers, cargo space, seats, Automatic or manual).

### LINKED ISSUE #103: [SUBTASK of story #81] Smooth navigation between list/grid view and detailed view.

### LINKED ISSUE #105: [SUBTASK of story #81] Test browsing experience on different browsers and devices for responsiveness and usability.

### PR DESCRIPTION
**Pull request by JeremyCrete**
### 🛠 Proposed Changes:

Created browse vehicle page, added filtering options and connected the cars to the database

### 📝 Details:

-Browse vehicle page is user friendly and allows you to view all relevant information about the vehicles

### 🔗 Related Issue(s):

- Resolves #99  
- Resolves #100 
- Resolves #102
- Resolves #101 
- Resolves #103 
- Resolves #105

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 pnpm-lock.yaml
 package-lock.json
 yarn.lock
+hooks.server.ts
```
Comment (NathanGrenier) [inline .prettierignore:5]: Why is this file being ignored when it comes to formatting?

File: src/lib/controllers/carController.ts
Comment (NathanGrenier) [inline src/lib/controllers/carController.ts:1]: Having a bunch of functions described in a file is greater if they obfuscate complexity. 

However, I think it's fine if you just use the prisma library directly in the page.server files if these functions are only being used once in a single place.

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -109,3 +131,33 @@
...
+
+  li {
+    margin-right: 20px;
+  }
+
+  nav ul li a {
+    display: inline-block;
+    padding: 10px 20px;
+    border-radius: 5px;
+    background-color: #08417a;
```
Comment (NathanGrenier) [inline src/routes/+layout.svelte:155]: Make sure to try and use the skeleton colors and fonts wherever possible.

Replace this with one of the variants or the predefined tailwind color classes.

File: src/routes/browse-vehicles/+page.svelte
Comment (NathanGrenier) [inline src/routes/browse-vehicles/+page.svelte:1]: Try an use the components provided by skeleton wherever possible. It'll make the site look more consistent.

File: src/routes/browse-vehicles/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  <span class="price-display">${filteredPrice}</span>
+</div>
+
+<div class="car-container">
+  {#each cars as car, index}
+    {#if (!filteredColor || car.colour === filteredColor) && (!filteredPrice || car.dailyPrice <= filteredPrice)}
+      <div class="car">
+        <div class="car-inner">
+          <!-- <img src={car.photoUrl} alt={car.make} /> -->
+          <img src={"link"} alt={"image of a car"} />
```
Comment (NathanGrenier) [inline src/routes/browse-vehicles/+page.svelte:48]: Replace `"link"` with an actual link

**Review by NathanGrenier — CHANGES_REQUESTED** [review]

**Review by CarciDev — APPROVED** [review]

**Pull request merged**